### DISCUSSION TITLE
`Failed to load version/info Error: Unsuccessful HTTP response`

### DISCUSSION THREAD

**Discussion by minty99** [argoproj/argo-workflows#6182] [Q&A]
I am using Argo Workflows v3.0.7.

In login UI, the error `Failed to load version/info Error: Unsuccessful HTTP response` occurs and the version number at the left top shows "unknown". How can I fix this error? It doesn't affect the features of Argo Workflows, but It doesn't seem to good.

**Comment by mseiwald** [comment]
I get the same error with `v3.1.1`. Although for me it makes the Argo UI unusable. When I click on anything, I'm immediately redirected back to the "Login" Page even though I have sso disabled.

**Comment by hseligson1** [comment]
I get this same error even though I've disabled auth mode and am using v3.1.13. It originally started when I tried to access the Workflow Templates and Cluster Workflow Templates. I received an error when I immediately deleted the metadata to add a new template.
So, I exited the UI and browser and terminated my CLI in case there was a caching issue. Now, I cannot login and I have an Unknown version in the top left-hand side under the Argo logo. When I try to click on anything within the UI I receive the: "Failed to load version/info Error: Unauthorized"

**Comment by pavmohan1** [comment]
Also, having this issue with SSO enabled, but with a visible version number. It is not affecting the usability of Workflows

**Comment by prein** [comment]
Same here. Happens to only one specific user while it works fine for other users. SSO (with Okta) enabled.
Would be great for the error to be more specific and meaningful.

**Reply by liujiong1982** [reply]
same here

**Reply by ltartarini** [reply]
same here.

**Reply by PaulSidze** [reply]
+1

**Comment by BonySmoke** [comment]
I faced this issue when I enabled SSO and RBAC for it. Once I disabled RBAC, the issue was solved.
I used image version `v3.3.8`.

**Reply by ltartarini** [reply]
Ok, but how can it be solved if RBAC is needed?

**Reply by BonySmoke** [reply]
That's a good question:)

**Comment by gogovan-vincentngai** [comment]
Any one know the issue fixed or not ?

**Comment by dayflower** [comment]
In my case, following [Service Account Secrets](https://argo-workflows.readthedocs.io/en/latest/service-account-secrets/) worked.
